plugins/woocommerce/tests/legacy/unit-tests/discounts/discounts-util.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Test for the discounts utility class.
+ * @package WooCommerce\Tests\DiscountsUtil
+ */
+
+use Automattic\WooCommerce\Utilities\DiscountsUtil;
+
+/**
+ * WC_Tests_DiscountsUtil.
+ */
+class WC_Tests_DiscountsUtil extends WC_Unit_Test_Case {
```

Comment:
Thank you for adding, but can we relocate (and rename) so this lives at:

```
tests/php/src/Utilities/DiscountsUtilTest.php
```

[(See guidance here.)](https://github.com/woocommerce/woocommerce/blob/8.9.1/plugins/woocommerce/tests/README.md#guide-for-writing-unit-tests)

Reply:
Thank you for pointing this out for me, I made the needed adjustments 👍🏼